frontend/src/components/FileTables/common/FileTable/TablePaginator.tsx

```diff
@@ -20,18 +25,45 @@ export const TablePaginator = () => {
   const pathname = usePathname();
   const searchParams = useSearchParams();
 
+  const [pageInput, setPageInput] = useState<string>((page + 1).toString());
```

Comment:
You have several `(page + 1).toString()`. I'd use a function for simplifying this.